ui/eureka/eureka.js

```diff
@@ -687,81 +593,63 @@ let Options = {
             ko: '택심',
             cn: '读书人',
           },
-          mobName: {
-            en: 'Taxim',
-            de: 'Taxim',
-            fr: 'Taxim',
-            ja: 'タキシム',
-            ko: '택심',
-            cn: '塔克西姆',
-          },
           trackerName: {
             en: 'Taxim',
             de: 'Taxim',
             fr: 'Taxim',
             ja: 'タキシム',
-            ko: '택심',
+            ko: '텍심',
             cn: '读书人',
           },
           x: 25.5,
           y: 28.3,
+          fateID: 1369,
+          bunny: false,
```

Comment:
I think "bunny: false" can just be implied when missing, and not included in every fate, if you don't mind removing them.